basics/getting_started.md

```diff
@@ -3,18 +3,22 @@ layout: contribute
 title: Getting Started
 ---
 
-# Setting up your coding environment
+# Getting started
 
-For now we have support for IntelliJ. We don't have IDE support for other
-languages in Bazel right now.
+This page helps you set up your coding environment, describes creating an
+IntelliJ project and takes you through compiling and debugging your project.
+Finally, you can read about Bazel's source code, how to navigate through it, and
+how to use the Bazel CI system.
 
-## Preparations
+## Installing Bazel
 
 *  [Install Bazel](https://bazel.build/versions/master/docs/install.html) on your
```

Comment:
1. Install the latest version of Bazel on your system. For instructions, see [Installing Bazel](https://docs.bazel.build/versions/master/install.html).

(Rationale for edits: Make the call to action/ clickable text last. Structuring this way tells users which version to install up front so they don't miss before they click. The URL in the existing page looks like an artifact from an older time and redirects to the link above.)

Reply:
Also, do we want to point users at the main install page, or should we point them at the Compiling Bazel from source page: https://docs.bazel.build/versions/master/install-compile-source.html?

Reply:
That question raises a concern with the install-compile-source page you pointed to. Most people (external and internal) will compile Bazel from source, but with an installed, pre-distributed bazel.
Far fewer will use the second half of that page, which describes bootstraping. I don't think I have ever needed the bootstrap. It is too unweildy.

OTOH..... there is an important group of people who want to compile bazel from a bootstrap source who will never actually modify bazel itself.   They would be the tool administrators for large corporations that have SOX compliance issues. They will want to bootstrap Bazel and provide their own version for all the users in their org. They, very likely, will not make local modifications or contribute patches.

It occurs to me now that we have not talked a lot about the different categories of users of the documentation. End users are obvious. Contributors are a subset of end users. Administrators are an important (yet very small) class of gate-keepers. They might be users, but not necessarily.

Reply:
Reworded, and linked to the Compiling Bazel from source page.